lib/opentok/broadcasts.rb

```diff
@@ -159,6 +159,101 @@ def layout(broadcast_id, options = {})
       (200..300).include? response.code
     end
 
+    # Adds a stream to currently running broadcast that was started with the
+    # streamMode set to "manual". For a description of the feature, see
+    # {https://tokbox.com/developer/rest/#selecting-archive-streams}.
+    #
+    # @param [String] archive_id
+    #   The broadcast ID.
+    #
+    # @param [String] stream_mode
+    #   The stream mode of the broadcast. Must be "manual" in order to add a stream.
+    #
+    # @option opts [String] :add_stream
+    #   The ID for the stream to be added to the broadcast
```

Comment:
Sure thing. I only named it that way to be in line with the JSON property names expected by the API end-point, but if it's going to be its own param rather than part of the options hash, then it makes sense to name it `stream_id`

Reply:
Sure. I think the original reasoning behind this is covered in my response to your comments on the `Archives` class, but I'm happy to change the implementation and move some functionality to the wrapper method.